pyprocessing/color.py

```diff
@@ -4,25 +4,40 @@
 
 
 class Color:
-    RGB = 0
-    HSV = 1
+    RGB = 1
     HLS = 2
+    HSB = 3
 
-    def __init__(self, *args, colorspace=0):
-        self.colorspace = colorspace
+    colorspace = RGB
+    maxv1 = maxv2 = maxv3 = maxva = 255
+
+    def __init__(self, *args):
```

Comment:
don't remove the colorspace kwarg, as defining a Color in a different colorspace than the default can be useful.

Just set it to None and add a condition

Reply:
I'd agree it's useful, but that's not in processing. I'd argue that should be as faithful to it as possiblem rather than adding improvements

Reply:
The goal is to provide an API that is pretty much copy and paste from Processing to PyProcessing, not necessarily the other way around. Having non disruptive improvements is fine, as it can also be useful for specifying colors in a specific colorspace for the library's own use (most GUI toolkits only understand the RGB colorspace, so it would be a pain to set the global colorspace to RGB and change it back every time we want to render something on screen, plus it's terribly innefficient)